src/app/pages/StoryPage/StoryPage.jsx

```diff
@@ -197,19 +208,20 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
     table: props => <CpsTable {...props} />,
     mpu: props =>
       isAdsEnabled ? <MpuContainer {...props} slotType="mpu" /> : null,
-    wsoj: props => (
-      <CpsRecommendations
-        {...props}
-        parentColumns={gridColsMain}
-        items={recommendationsInitialData}
-      />
-    ),
+    wsoj: props =>
+      service !== 'hindi' ? (
+        <CpsRecommendations
+          {...props}
+          parentColumns={gridColsMain}
+          items={recommendationsInitialData}
+        />
+      ) : null,
```

Comment:
We may need to keep this here as the `addExperimentPlaceholderBlocks` function in `getInitialData` will include the `wsoj` block type for all other services other than Hindi

Reply:
Had a look at this too. Problem is we do `addRecommendationsBlock` above `addExperimentPlaceholderBlocks` which also adds blocks with `wsoj` that calls the cpsRecommendation. Not a problem for control but it is a problem for split recs and the other variations as it will render both components (normal recs and split recs).

Reply:
For the experimentation blocks, was the code copied from `addExperimentPlaceholderBlocks`? I didn't remove it because wasn't too sure at the time; so I kept it and added that check.

Reply:
My bad, just saw your comment below regarding `addRecommendationsBlock`. I will remove it.

Reply:
Done 👍